server/storage/backend/verify.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	ENV_VERIFY_VALUE_LOCK verify.VerificationType = "lock"
+	EnvVerifyValueLock verify.VerificationType = "lock"
```

Comment:
Even though this is exported, I just renamed it as it is used only by the `backend_tests` package, which is in the same directory but in a different package.